instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
@smarnach @pomegranited 
I would prefer that you define the permission on the `InstanceReference` model, which is more or less a global model, and you create the permission using that model's `Meta`: https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#custom-permissions

No migration required (it creates the permission automatically during `post_migrate`).

I don't think you should create a custom model just to have a permission.